src/components/Interpretations/InterpretationsUnit/InterpretationList.js

```diff
@@ -9,10 +9,10 @@ const sortByDateProp = (a, b) => {
     const dateB = b.created
 
     if (dateA < dateB) {
```

Comment:
Maybe this function could be named `sortByCreatedDateDesc` ?

Reply:
Renamed.